functions/supplier/jdbc-supplier/src/test/java/org/springframework/cloud/fn/supplier/jdbc/DefaultJdbcSupplierTests.java

```diff
@@ -76,6 +82,13 @@ void testExtraction() {
 		stepVerifier.verify();
 	}
 
+	@Test
+	void verifyNoIntMessageGroupTable() {
+		assertThatExceptionOfType(BadSqlGrammarException.class)
```

Comment:
This is indirectly verifying the fact that the schema did not initialize. Might be helpful to add a 1-liner comment or include this fact in the name of the test.

Reply:
awesome details - thanks @artembilan